modules/artifact/src/main/java/org/mule/runtime/module/artifact/api/classloader/MuleArtifactClassLoader.java

```diff
@@ -322,6 +326,8 @@ private void shutdownListeners() {
    */
   protected ResourceReleaser createResourceReleaserInstance() {
     if (resourceReleaserInstance == null) {
+      createClass(threadGroupContextClassLoaderSoftReferenceBusterLocation);
```

Comment:
why is this here? nobody uses the generated class afterwards

Reply:
Because it is used internally by the resourceReleaser.